### DISCUSSION TITLE
Is this library slower or faster than fasthttp client?

### DISCUSSION THREAD

**Discussion by rew1nter** [go-resty/resty#526] [Q&A] [answered]
I understand it might not be the goal for you to make this a *fast* client. Just asking outta curiousity

**Comment by moorereason** [comment, marked as answer]
The [valyala fasthttp client](https://github.com/valyala/fasthttp) claims to be faster than the Go stdlib package, which Resty uses.

**Reply by rew1nter** [reply]
Fasthttp client has too many drawbacks.
So resty is a wrapper around stdlib and not a replacement. That's really nice to know.
